docs/reference/feast-cli-commands.md

```diff
@@ -24,6 +24,7 @@ Commands:
   init                     Create a new Feast repository
   materialize              Run a (non-incremental) materialization job to...
   materialize-incremental  Run an incremental materialization job to ingest...
+  permissions              Access permissions
```

Comment:
probably we can have an `rbac` command as well that is the same as `permissions`